java/system-test/system-test-drivers/src/main/java/sleeper/systemtest/drivers/metrics/AwsTableMetricsDriver.java

```diff
@@ -97,7 +97,7 @@ public TableMetrics getTableMetrics() {
                 .recordCount((long) getMetric(map, "records"))
                 .partitionCount((int) getMetric(map, "partitions"))
                 .leafPartitionCount((int) getMetric(map, "leafPartitions"))
-                .averageActiveFilesPerPartition(getMetric(map, "filesPerPartition"))
+                .averageFileReferencesPerPartition(getMetric(map, "filesReferencesPerPartition"))
```

Comment:
This will break, as it doesn't match the key in the METRIC_ID_TO_NAME map any more.